# Individual subscriber configuration:
#   name:           Source name
#   topic:          The topic that provides  messages
#   timeout:        Time in seconds without incoming messages to consider this topic inactive
#   priority:       Priority: an UNIQUE unsigned integer from 0 (lowest) to MAX_INT 
#   short_desc:     Short description (optional)


# Fin Angles
fin_angles_subscribers:
  - name:        "Autopilot"
    topic:       "input/autopilot/set_angles"
    timeout:     0.1
    priority:    0
    short_desc:  ""
  - name:        "Jaus Ros Bridge"
    topic:       "input/jausRosBridge/set_angles"
    timeout:     0.5
    priority:    1
    short_desc:  ""
fin_angles_publisher:       "fin_control/set_angles"


# Thruster
set_rpm_subscribers:
  - name:        "Autopilot"
    topic:       "input/autopilot/set_rpm"
    timeout:     0.1
    priority:    0
    short_desc:  ""
  - name:        "Jaus Ros Bridge"
    topic:       "input/jausRosBridge/set_rpm"
    timeout:     0.5
    priority:    1
    short_desc:  ""
set_rpm_publisher:       "thruster_control/set_rpm"